packages/node-cli/src/commands/ImportFilesCommandGenerator.js

```diff
@@ -177,35 +179,49 @@ module.exports = class ImportFilesCommandGenerator extends BaseCommandGenerator
 		return answers;
 	}
 
-	_executeAction(answers) {
-		if (this._projectInfoService.getProjectType() === PROJECT_SUITEAPP) {
-			throw TranslationService.getMessage(ERRORS.IS_SUITEAPP);
-		}
-
-		const executionContextImportObjects = new SDKExecutionContext({
-			command: this._commandMetadata.sdkCommand,
-			includeProjectDefaultAuthId: true,
-			params: answers,
-		});
+	async _executeAction(answers) {
+		try {
+			if (this._projectInfoService.getProjectType() === PROJECT_SUITEAPP) {
+				throw TranslationService.getMessage(ERRORS.IS_SUITEAPP);
+			}
 
-		return executeWithSpinner({
-			action: this._sdkExecutor.execute(executionContextImportObjects),
-			message: TranslationService.getMessage(MESSAGES.IMPORTING_FILES),
-		});
+			const executionContextImportObjects = new SDKExecutionContext({
```

Comment:
variable names should be nouns, not verbs (the verbs are for functions/methods). Can we rename to importObjectsExecutionContext?